src/WorkItems/AzureDevOpsClientWrapper.cs

```diff
@@ -13,21 +13,20 @@
 using Microsoft.VisualStudio.Services.WebApi.Patch.Json;
 using Newtonsoft.Json;
 
-namespace Microsoft.WorkItemFiling
+namespace Microsoft.WorkItems
 {
     /// <summary>
     /// Represents an Azure DevOps project in which work items can be filed.
     /// </summary>
-    public class AzureDevOpsClientWrapper : FilingClient
+    public class AzureDevOpsClientWrapper: FilingClient
     {
         private WorkItemTrackingHttpClient _witClient;
-        
-        public override async Task Connect(string personalAccessToken)
+        public override async Task Connect(string securityToken)
         {
             Uri accountUri = new Uri(this.AccountOrOrganization, UriKind.Absolute);
 
-            VssConnection connection = new VssConnection(accountUri, new VssBasicCredential(string.Empty, personalAccessToken));
-            await connection.ConnectAsync();
+            VssConnection connection = new VssConnection(accountUri, new VssBasicCredential(string.Empty, securityToken));
+            connection.ConnectAsync().Wait();
```

Comment:
>Wait [](start = 38, length = 4)

Just curious, do we have to block?

Reply:
+1 on this, since in the corresponding GitHub code, we actually do `await`. And yet this was an explicit change, so there must have been a reason... great place for a "why" comment.

---
In reply to: [381451518](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381451518) [](ancestors = 381451518)

Reply:
Just transient code condition that was useful while I was debugging. Removed.

---
In reply to: [381457917](https://github.com/microsoft/sarif-sdk/pull/1775#discussion_r381457917) [](ancestors = 381457917,381451518)